truss/templates/trtllm/packages/schema.py

```diff
@@ -84,14 +84,15 @@ def to_tensors(self):
 
 # The following are duplicated from the underlying base image.
 # We list them as a comment for posterity:
+# TODO(@joostinyi): Refactor to benefit from type sharing across serving image and truss lib
```

Comment:
curious, why do we need all this commented out code if most of it seems to also be in `config/trt_llm.py`?

Reply:
We leave these comments in case a user introspects the types we use the truss we generate for them, before we figure out how to make truss a dependency of this code to have clear references to these types.

Reply:
ah good to know. Thanks :+1: